skada/_dasvm.py

```diff
@@ -0,0 +1,225 @@
+# Author: Ruben Bueno <ruben.bueno@polytechnique.edu>
+# dasvm implementation
+
+import numpy as np
+import math
+
+from skada.base import BaseEstimator
+from skada.utils import check_X_y_domain, source_target_split
+from sklearn.base import clone
+
+from sklearn.svm import SVC
+
+
+class DASVMEstimator(BaseEstimator):
```

Comment:
You have to describe better the docstings. Link to the paper, parameters ... Look at other methods to see what's you have to do

Reply:
yes, I have done it, I just haven't pushed it yet, waiting for more important changes